packages/run/package.json

```diff
@@ -67,7 +67,7 @@
     "postpack": "rm -f oclif.manifest.json",
     "posttest": "tslint -p test -t stylish",
     "prepack": "rm -rf lib && tsc -b && oclif-dev manifest && oclif-dev readme",
-    "test": "nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\"",
```

Comment:
any resaon to remove --forbid-only?

Reply:
nope! that's a good catch. leftover from when i was using `.only` during dev. i'll fix